/**:
  ros__parameters:
    can_assign_matrix:
      #UNKNOWN, CAR, TRUCK, BUS,  TRAILER, MOTORBIKE, BICYCLE,PEDESTRIAN
      [0,       0,   0,     0,    0,       0,         0,      0,         #UNKNOWN
       0,       1,   1,     1,    1,       0,         0,      0,         #CAR
       0,       1,   1,     1,    1,       0,         0,      0,         #TRUCK
       0,       1,   1,     1,    1,       0,         0,      0,         #BUS
       0,       1,   1,     1,    1,       0,         0,      0,         #TRAILER
       0,       0,   0,     0,    0,       1,         1,      1,         #MOTORBIKE
       0,       0,   0,     0,    0,       1,         1,      1,         #BICYCLE
       0,       0,   0,     0,    0,       1,         1,      1]         #PEDESTRIAN

    max_dist_matrix:
      #UNKNOWN, CAR, TRUCK, BUS,  TRAILER, MOTORBIKE, BICYCLE, PEDESTRIAN
      [4.0,     4.0, 5.0,   5.0,  5.0,     2.0,       2.0,     2.0,       #UNKNOWN
       4.0,     2.0, 5.0,   5.0,  5.0,     1.0,       1.0,     1.0,       #CAR
       5.0,     5.0, 5.0,   5.0,  5.0,     1.0,       1.0,     1.0,       #TRUCK
       5.0,     5.0, 5.0,   5.0,  5.0,     1.0,       1.0,     1.0,       #BUS
       5.0,     5.0, 5.0,   5.0,  5.0,     1.0,       1.0,     1.0,       #TRAILER
       2.0,     1.0, 1.0,   1.0,  1.0,     3.0,       3.0,     3.0,       #MOTORBIKE
       2.0,     1.0, 1.0,   1.0,  1.0,     3.0,       3.0,     3.0,       #BICYCLE
       2.0,     1.0, 1.0,   1.0,  1.0,     3.0,       3.0,     2.0]       #PEDESTRIAN
    max_rad_matrix: # If value is greater than pi, it will be ignored.
      #UNKNOWN, CAR,   TRUCK, BUS,    TRAILER MOTORBIKE, BICYCLE, PEDESTRIAN
      [3.150,   3.150, 3.150, 3.150,  3.150,  3.150,     3.150,   3.150,      #UNKNOWN
       3.150,   1.047, 1.047, 1.047,  1.047,  3.150,     3.150,   3.150,      #CAR
       3.150,   1.047, 1.047, 1.047,  1.047,  3.150,     3.150,   3.150,      #TRUCK
       3.150,   1.047, 1.047, 1.047,  1.047,  3.150,     3.150,   3.150,      #BUS
       3.150,   1.047, 1.047, 1.047,  1.047,  3.150,     3.150,   3.150,      #TRAILER
       3.150,   3.150, 3.150, 3.150,  3.150,  3.150,     3.150,   3.150,      #MOTORBIKE
       3.150,   3.150, 3.150, 3.150,  3.150,  3.150,     3.150,   3.150,      #BICYCLE
       3.150,   3.150, 3.150, 3.150,  3.150,  3.150,     3.150,   3.150]      #PEDESTRIAN

    min_iou_matrix: # If value is negative, it will be ignored.
      #UNKNOWN, CAR,   TRUCK, BUS,    TRAILER,  MOTORBIKE, BICYCLE, PEDESTRIAN
      [0.1,     0.1,   0.1,   0.1,    0.1,      0.1,       0.1,     0.1,       #UNKNOWN
       0.1,     0.3,   0.2,   0.2,    0.2,      0.1,       0.1,     0.1,       #CAR
       0.1,     0.2,   0.3,   0.3,    0.3,      0.1,       0.1,     0.1,       #TRUCK
       0.1,     0.2,   0.3,   0.3,    0.3,      0.1,       0.1,     0.1,       #BUS
       0.1,     0.2,   0.3,   0.3,    0.3,      0.1,       0.1,     0.1,       #TRAILER
       0.1,     0.1,   0.1,   0.1,    0.1,      0.1,       0.1,     0.1,       #MOTORBIKE
       0.1,     0.1,   0.1,   0.1,    0.1,      0.1,       0.1,     0.1,       #BICYCLE
       0.1,     0.1,   0.1,   0.1,    0.1,      0.1,       0.1,     0.1]       #PEDESTRIAN
